src/test/java/picard/sam/markduplicates/MarkDuplicatesWithMateCigarTest.java

```diff
@@ -37,6 +37,16 @@ protected AbstractMarkDuplicatesCommandLineProgramTester getTester() {
         return new MarkDuplicatesWithMateCigarTester();
     }
 
+    /**
+     * Currently this class fails this test due to
+     * a small difference in the selection of representative read. To fix this
+     * a change is required in htsjdk.samtools.DuplicateScoringStrategy.compare
```

Comment:
Create a github issue for this? Or is it not worth fixing?

Reply:
done.